src/TransformBase.cs

```diff
@@ -225,8 +225,16 @@ private string GetParameterTypeName(Type type)
                         type.GetGenericTypeDefinition().FullName ?? string.Empty :
                         type.GetGenericTypeDefinition().Name;
 
-                    sb.Append(genericName);
-                    sb.Append('[');
+                    if (genericName.Contains(Constants.GenericParameterBackTick))
+                    {
+                        sb.Append(genericName.Remove(genericName.IndexOf(Constants.GenericParameterBackTick)));
```

Comment:
Would be nice if you can add a comment here clarifying this is to skip the ``"`2"`` part of something like ``Dictionary`2``.